### DISCUSSION TITLE
Using Pytorch Frame in assigning node features

### DISCUSSION THREAD

**Discussion by parsisoftware** [pyg-team/pytorch_geometric#8376] [Q&A]
Hello,
Can you please guide me how to use the PyTorch frame encodings (https://github.com/pyg-team/pytorch-frame/tree/master) for the node features while loading the data from csv using the following link:
 https://pytorch-geometric.readthedocs.io/en/latest/tutorial/load_csv.html

Can I simply replace the encoding classes with the one defined in the above link like the following:

from torch_frame import stype
from torch_frame.nn import (
EmbeddingEncoder,
FTTransformer,
LinearEmbeddingEncoder,
LinearEncoder,
)

movie_x, movie_mapping = load_node_csv(
movie_path, index_col='movieId', encoders={
'title': LinearEmbeddingEncoder(),
'genres': EmbeddingEncoder()
})

**Comment by rusty1s** [comment]
We are working on examples and some tutorials right now. The current idea is to store `TensorFrame` from `pytorch-frame` as attribute in `Data`. You can then encode this `TensorFrame` using `pytorch-frame`, and pass it further down to PyG.

**Reply by denadai2** [reply]
this would be awesome, especially if we can use it in our gnn models

**Reply by rusty1s** [reply]
We have pushed an example in https://github.com/snap-stanford/relbench, see https://github.com/snap-stanford/relbench/blob/main/relbench/external/graph.py and https://github.com/snap-stanford/relbench/blob/main/relbench/external/nn.py. This might be helpful. However, we still work on a bringing an example to PyG directly.

**Reply by parsisoftware** [reply]
When the example in PyG will be ready?
